File: Beecrowd/1176 Fibonacci em Vetor/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        final int TAM = 61;
        int n, t;
        long[] fibonacci = new long[TAM];
        long ant = 0, prox = 1;

        for (int i = 0; i <= 60; i++)
            if (i < 2)
                fibonacci[i] = ant + i;
            else {
                fibonacci[i] = ant + prox;
                ant = prox;
                prox = fibonacci[i];
            }

        t = sc.nextInt();

        for (int i = 0; i < t; i++) {
            n = sc.nextInt();
            System.out.println("Fib(" + n + ") = " + fibonacci[n]);
        }

        sc.close();
    }

}

File: Beecrowd/1176 Fibonacci em Vetor/main.py
j = 0
T = int(input())

while T > j:
  N = int(input())
  ant = 0 
  prox = 1

  for i in range(N + 1):
    if i < 2:
      fibonacci = ant + i
    else:
      fibonacci = ant + prox
      ant = prox
      prox = fibonacci
  print(f"Fib({N}) = {fibonacci}")
  j += 1